src/components/EditCoursePage/EditCourseForm.jsx

```diff
@@ -120,7 +141,8 @@ export class BaseEditCourseForm extends React.Component {
           <Collapsible
             title={this.formatCourseTitle(title, courseStatuses)}
             key="Test Key"
-            isOpen
+            isOpen={this.state.open}
```

Comment:
Up to you, but I might deconstruct this. If we are only using the state in this one spot and with one variable, not totally necessary though.